orders/views.py

```diff
@@ -83,3 +103,58 @@ def delete(self, request, pk):
             return HttpResponse(status = 200)
         except OrderItem.DoesNotExist:
             return HttpResponse(status = 400)
+
+class OrderCompletedView(View):
+    @check_authorization
+    def post(self, request):
+        try:
+            if not Order.objects.filter((Q(customer = request.user.id)& Q(order_status = UNSETTLED))).exists():
+                return HttpResponse(status = 400)
+            data                      = json.loads(request.body)
+            order                     = Order.objects.get((Q(customer = request.user.id)& Q(order_status = UNSETTLED)))
+            email                     = data.get('email')
+            order.order_status_id     = SETTLEMENT
+            order.number              = str(order.updated_at) + str(request.user.id)
+            order.total_price         = data.get('total_price')
+            order.order_full_name     = data.get('full_name')
+            order.payment_method      = data.get('payment')
+            order.customer_address    = data.get('address')
+            order.save()
+            Order (
+                customer = request.user,
+                order_status_id = UNSETTLED,
+                    ).save()
```

Comment:
들여쓰기 포맷 공통적으로 수정해야할 것 같습니다.
```suggestion
            Order(
                customer = request.user,
                order_status_id = UNSETTLED,
            ).save()
```